onset_maproom/calc.py

```diff
@@ -34,38 +33,36 @@ def water_balance(
     sminit,
     time_coord="T",
 ):
-    """ Calculate soil moisture.
-
-        Estimates soil moisture from:
-            Rainfall, evapotranspiration, total available water and intial soil moisture value. 
-        Knowing that:
-            sm(t) = sm(t-1) + rain(t) - et(t)
-        With roof and floor respectively at taw and 0 at each time step.
-
-        Parameters
-        ------
-        daily_rain : DataArray
-                    Daily rainfall data.
-        et : int
-                    Evapotranspiration (constant).
-        taw : int
-                    Total available water (consant).
-        sminit : int
-                    Soil moisture initialization (constant).
-        time_coord : str, optional             
-                    Time grid in 'daily_rain' (the default is "T", which implies
-                    naming convention of time grid as "T") 
-        Returns
-        -------
-        water_balance : Dataset
-                    Daily soil moisture data.
-        See Also
-        --------
-        Notes
-        -----
-        Examples
-        --------
-
+    """Calculate soil moisture.
+
+    Estimates soil moisture from:
+        Rainfall, evapotranspiration, total available water and intial soil moisture value. 
+    Knowing that:
+        `water_balance`(t) = `water_balance`(t-1) + `daily_rain`(t) - `et`(t) 
```

Comment:
it should rather be `soil_moisture`(t) = `soil_moisture`(t-1)...
water_balance is the dataset that contains soil_moisture. It might contain more outputs in the future.